### PR TITLE
OSDOCS-5812-11:vSphere/VMC sample YAML removed text from callout

### PR DESCRIPTION
**Pull request by dfitzmau**
* Fixed formatting issues with text running into the navigation menu in [original doc](https://docs.openshift.com/container-platform/4.11/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-vsphere-config-yaml_installing-vsphere-network-customizations)

[OSDOCS-5812](https://issues.redhat.com/browse/OSDOCS-5812)

Version(s):
4.11

Link to docs preview:

vSphere:

* [Installing a cluster on vSphere with user-provisioned infrastructure](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere)
* [Installing a cluster on vSphere with network customizations](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-vsphere-config-yaml_installing-vsphere-network-customizations)
* [Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-vsphere-config-yaml_installing-restricted-networks-vsphere)

VMC:
* [Installing a cluster on VMC with user-provisioned infrastructure](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-vsphere-config-yaml_installing-vmc-user-infra)
* [Installing a cluster on VMC in a restricted network with user-provisioned infrastructure](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-restricted-networks-vmc-user-infra.html#installation-vsphere-config-yaml_installing-restricted-networks-vmc-user-infra)
* [Installing a cluster on VMC with user-provisioned infrastructure and network customizations](https://59197--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations-user-infra.html#installation-vsphere-config-yaml_installing-vmc-network-customizations-user-infra)

QE review:
- [X] QE has approved this change.

Additional information:
[Slack](https://redhat-internal.slack.com/archives/C04T569EL1Z/p1682079305220229?thread_ts=1682071293.692209&cid=C04T569EL1Z)

### CODE REVIEW

**Comment by dfitzmau** [comment]
> /lgtm, compared with 4.12 PR, this PR have some extra format changes, if it's on purpose, I'm OK, thanks.

Hi @sgaoshang . Thanks for asking. I've had to remove some whitespace that is causing the following formatting issue: [Published section in 4.11 doc](https://docs.openshift.com/container-platform/4.11/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere)

I'll apply the same formatting changes to 4.12.

File: modules/installation-vsphere-config-yaml.adoc [hunk 1]
```diff
@@ -103,48 +103,24 @@ imageContentSources: <18>
...
-<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
+<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/ optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/ example_datacenter/host/example_cluster/Resources`.
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:123]:
```suggestion
<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
```
I'm not sure why the `<` got removed here, but it should be re-added. there are a couple of incorrect spaces as well.

File: modules/installation-vsphere-config-yaml.adoc [hunk 2]
```diff
@@ -103,48 +103,24 @@ imageContentSources: <18>
...
-value must match the number of control plane machines that you deploy.
+<4> You must set the value of the `replicas` parameter to `0`. This parameter controls the number of workers that the cluster creates and manages for you, which are functions that the cluster does not perform when you use user-provisioned infrastructure. You must manually deploy worker machines for the cluster to use before you finish installing {product-title}.
+<5> The number of control plane machines that you add to the cluster. Because the cluster uses this values as the number of etcd endpoints in the cluster, the value must match the number of control plane machines that you deploy.
 <6> The cluster name that you specified in your DNS records.
 <7> The fully-qualified hostname or IP address of the vCenter server.
-<8> The name of the user for accessing the server. This user must have at least
-the roles and privileges that are required for
-link:https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/vcp-roles.md[static or dynamic persistent volume provisioning]
-in vSphere.
+<8> The name of the user for accessing the server.
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:118]: is there a reason why the additional information with the link was removed here?

**Review by bscott-rh — APPROVED** [review]
2 questions, otherwise LGTM.

**Pull request merged**